javascript/javascript1/week2/flight_booking.js

```diff
@@ -0,0 +1,29 @@
+function getFullname(firstname,surname,useFormalName,gender){
+    if(firstname===' ' || surname===' '){
```

Comment:
Try and test with names like `""` (empty string - length 0).

I would recommend you start protecting against that. If you wanna go the extra mile and protect against a name with just a single space `" "` then why not 2 spaces or 3 spaces (`"  "` or `"   "`). If you start going down that rabbit hole I would recommend looking at trim (string trimming).